articles/connections/social/google.md

```diff
@@ -14,6 +14,8 @@ seo_alias: google
 
 To connect your Auth0 client to Google and Google Apps, you will need to generate a *Client ID* and *Client Secret* in a Google project, copy these keys into your Auth0 settings, and enable the Connection.
 
+If you are looking to manage authentication in your native Android app [see the Android quickstart](/quickstart/native/android).
+
```

Comment:
The number of steps will probably vary per connection doc, so it's not always 10. I would leave the paragraph title at the main doc, and add the content+links at the fragment doc.